docker-compose.test.yml

```diff
@@ -21,7 +21,7 @@ services:
     volumes:
       - ./:/var/www/api
       - ./data/media/incoming:/data/media/incoming
-      - ./data/media/audio:/data/media/audio
+      - ./data/media/audio:/test/media/audio
```

Comment:
This specifically needs to be `/data/media/audio`. Since this is the media _inside_ of the test container, we don't need to worry about it clashing with the "real data" folder (which, is the first half of this statement `host:container`).  Also we want it to be the same route in the test container as in the normal containers, otherwise we have to add special code for test containers, which seems to defeat the purpose of containerization somewhat.